UPGRADE.md

```diff
@@ -329,6 +329,8 @@ or decorate the `contao.intl.countries` service instead.
 To add or remove countries, you can use the `contao.intl.countries` configuration. `$GLOBALS['TL_LANG']['CNT']` can
 still be used for overwriting translations, but no longer to retrieve country names.
 
+Country codes have been changed to upper case letters.
```

Comment:
Maybe we could add an example on how to change it to lowercase?

https://www.php.net/manual/en/function.array-change-key-case.php

```php
$countries = System::getContainer()->get('contao.intl.countries')->getCountries();
$countries = array_change_key_case($countries);
```

Reply:
I think it's enough to give a hint here - I didn't have this on my table at all and hadn't read it in the news [https://contao.org/de/news/contao-5-0-ist-da](https://contao.org/de/news/contao-5-0-ist-da) ...

Reply:
This would most likely be in the upgrade.md and not the news since normal users do not care about this and explaining every change between 4 and 5 would be at least 400 pages long.

https://github.com/contao/contao/blob/5.x/UPGRADE.md#countriesphp-getcountries-and-globalstl_langcnt